### PR TITLE
Irons out a few kinks left by latest reftest improvements

### PR DESCRIPTION
**Pull request by AltGr**

### CODE REVIEW

**Comment by AltGr** [comment]
The windows test failure is I think due to a stale cache ?

**Comment by rjbou** [comment]
the windows failure is a regexp error:
```
Fatal error: exception Failure("Bad POSIX regexp: \"^NV_VARS|^OPAM_SWITCH_PREFIX|${OPAM}\"")
```

File: master_changes.md [hunk 1]
```diff
@@ -172,7 +172,7 @@ users)
   * Add switch-invariant test [#4866 @rjbou]
   * opam root version: add local switch cases [#4763 @rjbou] [2.1.0~rc2 #4715]
   * opam root version: add reinit test casess [#4763 @rjbou] [2.1.0~rc2 #4750]
-  * Add & update env tests [#4861 #4841 @rjbou @dra27]
+  * Add & update env tests [#4861 #4841 #4974 @rjbou @dra27 @AltGr]
```
Comment (rjbou) [inline master_changes.md:175]: Can you add reverts (remove or put a line on them), and add the engine updates ?

**Comment by AltGr** [comment]
Now it's OK on Windows but broken on Mac :sweat:

**Comment by AltGr** [comment]
Finally :tada: it's green :sweat_smile:

**Review by kit-ty-kate — APPROVED** [review]
LGTM

**Pull request merged**